docs/web-vitals.md

```diff
@@ -0,0 +1,295 @@
+# Web Vitals
+
+By default, Honeycomb includes events for all web vitals except [first-input-delay](https://web.dev/articles/fid). First-input-delay is set to be replaced by interaction-to-next-paint as a core web vital in March 2024.
+
+## Initialize Instrumentation
+
+To initialize this instrumentation, add it to the list of instrumentations.
+
+```js
+import { HoneycombWebSDK, WebVitalsInstrumentation } from '@honeycombio/opentelemetry-web'
+
+const sdk = new HoneycombWebSDK({
+  serviceName: 'my-app',
+  instrumentations: [new WebVitalsInstrumentation()],
+});
+```
+
+## Vitals Attributes
+
+All vitals have the following attributes, they will each be namespaced by the name of the vital:
+
+```ts
+/**
+ * The name of the vital (in acronym form).
+ */
+"<vital-name>.name": 'CLS' | 'FCP' | 'INP' | 'LCP' | 'TTFB';
+
+/**
+ * The rating as to whether the metric value is within the "good",
+ * "needs improvement", or "poor" thresholds of the metric.
+ */
+"<vital-rating>.rating": "good" | "needs-improvement" | "poor"
+
+/**
+ * A unique ID representing this particular metric instance. This ID can
+ * be used by an analytics tool to dedupe multiple values sent for the same
```

Comment:
should we scrap this comment? it was pulled from the generic web-vitals documentation, and honeycomb won't do any of the magic described — deduping or grouping multiple deltas together to caluculate a total.